### PR TITLE
[CELEBORN-861][BUILD] exclude `netty-handler-ssl-ocsp` for sbt

### PR DESCRIPTION
**Pull request by cfmcgrady**
### What changes were proposed in this pull request?

As title

### Why are the changes needed?

exclude `netty-handler-ssl-ocsp` for sbt like what we do for maven in https://github.com/apache/incubator-celeborn/pull/1544

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Pass GA

### CODE REVIEW

File: project/CelebornBuild.scala [hunk 1]
```diff
@@ -202,7 +202,7 @@ object CelebornCommon {
         "io.dropwizard.metrics" % "metrics-core" % metricsVersion,
         "io.dropwizard.metrics" % "metrics-graphite" % metricsVersion,
         "io.dropwizard.metrics" % "metrics-jvm" % metricsVersion,
-        "io.netty" % "netty-all" % nettyVersion,
+        ("io.netty" % "netty-all" % nettyVersion).exclude("io.netty", "netty-handler-ssl-ocsp"),
```
Comment (cfmcgrady) [inline project/CelebornBuild.scala:205]: I intend to consolidate all dependencies into a global object called `Dependencies`, similar to Maven's `dependencyManagement`, to improve dependency management. cc @pan3793

**Comment by cfmcgrady** [comment]
the diff between sbt and maven after this PR.

```diff
diff --git a/maven-files.out b/sbt-files.out
index fa4de49..7fb6567 100644
--- a/maven-files.out
+++ b/sbt-files.out
@@ -1,7 +1,6 @@
 ./celeborn-client-spark-3-shaded_2.12-0.4.0-SNAPSHOT.jar
-./META-INF/DEPENDENCIES
+./META-INF/INDEX.LIST
 ./META-INF/io.netty.versions.properties
-./META-INF/LICENSE
 ./META-INF/LICENSE.txt
 ./META-INF/MANIFEST.MF
 ./META-INF/maven/com.google.guava/guava/pom.properties
@@ -68,23 +67,12 @@
 ./META-INF/maven/io.netty/netty-transport-sctp/pom.xml
 ./META-INF/maven/io.netty/netty-transport-udt/pom.properties
 ./META-INF/maven/io.netty/netty-transport-udt/pom.xml
-./META-INF/maven/org.apache.celeborn/celeborn-client_2.12/pom.properties
-./META-INF/maven/org.apache.celeborn/celeborn-client_2.12/pom.xml
-./META-INF/maven/org.apache.celeborn/celeborn-client-spark-3_2.12/pom.properties
-./META-INF/maven/org.apache.celeborn/celeborn-client-spark-3_2.12/pom.xml
-./META-INF/maven/org.apache.celeborn/celeborn-client-spark-3-shaded_2.12/pom.properties
-./META-INF/maven/org.apache.celeborn/celeborn-client-spark-3-shaded_2.12/pom.xml
-./META-INF/maven/org.apache.celeborn/celeborn-client-spark-common_2.12/pom.properties
-./META-INF/maven/org.apache.celeborn/celeborn-client-spark-common_2.12/pom.xml
-./META-INF/maven/org.apache.celeborn/celeborn-common_2.12/pom.properties
-./META-INF/maven/org.apache.celeborn/celeborn-common_2.12/pom.xml
 ./META-INF/maven/org.apache.commons/commons-lang3/pom.properties
 ./META-INF/maven/org.apache.commons/commons-lang3/pom.xml
 ./META-INF/maven/org.jctools/jctools-core/pom.properties
 ./META-INF/maven/org.jctools/jctools-core/pom.xml
 ./META-INF/native/liborg_apache_celeborn_shaded_netty_transport_native_epoll_aarch_64.so
 ./META-INF/native/liborg_apache_celeborn_shaded_netty_transport_native_epoll_x86_64.so
-./META-INF/NOTICE
 ./META-INF/NOTICE.txt
 ./META-INF/services/reactor.blockhound.integration.BlockHoundIntegration
 ./org/apache/celeborn/client/ApplicationHeartbeater$$anon$1.class
@@ -802,6 +790,8 @@
 ./org/apache/celeborn/common/write/SimplePushStrategy.class
 ./org/apache/celeborn/common/write/SlowStartPushStrategy$CongestControlContext.class
 ./org/apache/celeborn/common/write/SlowStartPushStrategy.class
+./org/apache/celeborn/package$.class
+./org/apache/celeborn/package.class
 ./org/apache/celeborn/reflect/DynClasses$1.class
 ./org/apache/celeborn/reflect/DynClasses$Builder.class
 ./org/apache/celeborn/reflect/DynClasses.class
```

**Comment by pan3793** [comment]
Many Netty deps are needed to exclude, would prefer to build depList first.

**Comment by cfmcgrady** [comment]
addressed in https://github.com/apache/incubator-celeborn/pull/1831

**Pull request closed without merging**